File: Demo/app/src/main/java/com/example/mhamed/demo/vo/Category.kt
package com.example.mhamed.demo.vo

import com.example.mhamed.demo.model.Category

/**
 * @author mhamed
 * @since 2018-11-10
 */
data class Category(var slug: String, override var title: String,
                    override var description: String?, var active: Boolean) : Category {
}

File: QuickSeriesTestApp/QuickSeriesTestApp/Model/Category.swift
//
//  Category.swift
//  QuickSeriesTestApp
//
//  Created by Kasra Abasi on 5/18/19.
//  Copyright © 2019 noxowl. All rights reserved.
//

import Foundation

// this model is a simple one and is not a good fit for ViewModel

struct Category: Decodable {
    let title: String
    let description: String?
    let slug: String
}
